src/util.js

```diff
@@ -406,19 +406,32 @@
    */
   connect.PopupManager = function () { };
 
-  connect.PopupManager.prototype.open = function(url, name) {
+  connect.PopupManager.prototype.open = function (url, name, options) {
     var then = this._getLastOpenedTimestamp(name);
     var now = new Date().getTime();
-    var win = null;      
+    var win = null;
     if (now - then > ONE_DAY_MILLIS) {
-       win = window.open('', name);
-       if (win.location !== url) {
+      if (options && options.forceWindow === true) {
+        // default values below are chosen to provide a minimum height without scrolling
+        // and a unform margin based on the css of the ccp login page
+        var height = options.height ? options.height : 578;
```

Comment:
I am not too comfortable with hard-coding the width and height, but I understand there's no good option. @ctwomblyamzn do you know if we expose these information somehow in the code?

Reply:
Yeah, instead of hard-coding height and width, can we group these height and width variables and check whether they were supplied? If they aren't, go with window.open's default?

Reply:
`window.open()` defaults to opening in a new tab. That was the whole issue. So unfortunately not.

Reply:
Ah ok, sorry about the confusion. I don't see a better option than having defaults when `forceWindow` is true and no height parameter is supplied.

Reply:
could you assign these hard coded variable as a class variable. `var DEFAULT_WINDOW_HEIGHT=578`